apsuite/commisslib/meas_touschek_lifetime.py

```diff
@@ -77,17 +80,23 @@ class MeasTouschekLifetime(_BaseClass):
     RFFEAttSB = 30  # [dB] Singlebunch Attenuation
     FILTER_OUTLIER = 0.2  # Relative error data/fitting
 
-    # # calibration curves measured during machine shift in 2021/09/21:
+    # # calibration curves measured during machine studies shift in 2021/09/21:
```

Comment:
I think we can delete these old calibration curves...

Reply:
Ok, done.